documentation/source/theory/rmg/dynamics.rst

```diff
@@ -0,0 +1,96 @@
+.. _dynamics:
+
+*******************
+Dynamics Criterion
+*******************
+
+When dealing with more complex chemical mechanisms the standard RMG flux criterion
+has trouble picking up key chain branching reactions and has limited guarantees
+that it accurately represents the concentrations of all species.  The dynamics
+criterion is a measure of how much a given reaction affects core concentrations.
+This allows it to pick up key low flux chain branching reactions and better represent
+species concentrations.
+
+Calculating the Dynamics Criterion
+==================================
+Let us define production and consumption rates for a given species
```

Comment:
Even though it is probably obvious, I would put the symbols you use for production and consumption in this line so it is clear what you are saying. ("Let us define production (P_i(t)) and consumption (L_i(t)) rates for a given species")